src/dummy/WebsiteTask.cpp

```diff
@@ -0,0 +1,100 @@
+#include "WebsiteTask.hpp"
+#include "Arduino.h"
+#include "main.hpp"
+
+/* This is the data for the actual web page. */
+const static char quail_webpage[] = "<html> \
+<head><title>Basic webpage</title></head> \
+<body> \
+<h1> Welcome to your basic webpage hosted from Quail over Ethernet!</h1> \
+<img src=\"https://s3-us-west-1.amazonaws.com/habmc/assets/Logo_BlackSubText.png\"  style=\"width: 542px; height: 542px;\"> \
+<form action=\"i_can_haz_data\"> \
+  <input type=\"submit\" value=\"moar data\"> \
+</form> \
+<form action=\"too_much_data_stomp\"> \
+  <input type=\"submit\" value=\"stomp data\"> \
+</form> \
+</body> \
+</html>";
+const static char http_html_hdr[]   = "HTTP/1.0 200 OK\r\nContent-type: text/html\r\n\r\n";
+
+void WebsiteTask::activity()
+{
+	vTaskDelay(2000);
+	struct netbuf *inbuf;
+	char *         rq;
+	unsigned portSHORT len;
+	int                conn_check;
+
+	struct netconn *conn_1, *newconn_1;
+	/* create a connection structure */
+	conn_1 = netconn_new(NETCONN_TCP);
+	/* bind the connection to port on any IP address */
+	conn_check = netconn_bind(conn_1, NULL, 80);
+	while (conn_check != ERR_OK) {
+		LWIP_DEBUGF(LWIP_DBG_ON, ("Bind error=%d\n", conn_check));
+		goto conn_close;
+	}
+
+	/* tell the connection to listen for incoming
+	connection requests */
+    netconn_listen(conn_1);
+
+	for (;;) {
+
+		conn_check = netconn_accept(conn_1, &newconn_1);
```

Comment:
I'm confuzzled. Why can't you netcon_recv(conn_1, &inbuf)? What's the purpose of newconn_1?

Reply:
If we want to send and recieve over ethernet do we need four netconns? The SensorTask gets away with just one for sending.

Reply:
I think because you want to keep conn_1 open for listening and then have newconn for transferring the connection to and then actually servicing the requests

Reply:
hmm still slightly confused but ill read up